docs/source/release-process.rst

```diff
@@ -201,14 +201,12 @@ Once we are happy, push the updated docs to the GitHub ``gh-pages`` branch.
 Uploading conda packages
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-Travis and AppVeyor build zipline conda packages for us.  Once they have built
-and uploaded to anaconda.org the packages (and their dependencies) for the
-release commit to master, we should move those packages from the "ci" label to
-the "main" label.  You can do this from the anaconda.org web interface.  This
-is also a good time to remove all the old "ci" packages from anaconda.
-
-Travis and AppVeyor only build and upload linux-64 and win-64 packages.  We'll
-need to build and upload osx-64 packages manually on an OSX machine.
+Travis and AppVeyor build zipline conda packages for us (for Linux/OSX and
+Windows respectively).  Once they have built and uploaded to anaconda.org the
+zipline packages (and packages for their dependencies) for the release commit
+to master, we should move those packages from the "ci" label to the "main"
+label.  You can do this from the anaconda.org web interface.  This is also a
```

Comment:
Should we put a link to the services here? 

- https://anaconda.org/Quantopian/repo
- https://travis-ci.org/quantopian/zipline
- https://ci.appveyor.com/project/quantopian/zipline/branch/master

Figured having links in multiple places might be nice.